config/environments/test.rb

```diff
@@ -30,10 +30,13 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  config.action_mailer.default_options = { sender: 'test@test.com' }
 
```

Comment:
These enable the specs to run without relying on the real-life mail data that is configured separately (and not checked into the repo) in `application.yml`.